pandas/core/series.py

```diff
@@ -2743,17 +2743,20 @@ def nlargest(self, n=5, keep='first'):
 
         Parameters
         ----------
-        n : int
-            Return this many descending sorted values
-        keep : {'first', 'last'}, default 'first'
-            Where there are duplicate values:
-            - ``first`` : take the first occurrence.
-            - ``last`` : take the last occurrence.
+        n : int, default 5
+            Return this many descending sorted values.
+        keep : str, default 'first'
```

Comment:
I think it's better to keep `{'first', 'last', 'all'}`, as I don't think there is any other value allowed. That applies to both docstrings.